test/models/authorization/lifecycle_environment_authorization_test.rb

```diff
@@ -49,6 +53,10 @@ def test_promotable
     def test_any_promotable?
       refute KTEnvironment.any_promotable?
     end
+
+    def test_deletable
+      assert_empty KTEnvironment.deletable
+    end
```

Comment:
are these two 'test_deletable' methods actually testing different things?

Reply:
maybe you meant to test editable with one of them?

Reply:
It looks to me like each of those uses a different user - 1 is the "admin" user and the other is the "restricted" user.

So in that sense they are testing different user permissions to those sets of LCEs.

But probably would be good to add tests for editable, since I added that method too.